open_schools_platform/organization_management/circles/urls.py

```diff
@@ -10,7 +10,7 @@
     path('/ical', CirclesICalExportApi.as_view(), name='circles-ical'),
     path('/<uuid:pk>', MultipleViewManager({'get': GetCircleApi, 'delete': CircleDeleteApi}).as_view(), name='circle'),
     path('/<uuid:pk>/queries', CirclesQueriesListApi.as_view(), name='queries-list'),
-    path('/<uuid:pk>/students', CirclesStudentsListApi.as_view(), name='students-list'),
+    path('/<uuid:circle_id>/students', CirclesStudentsListApi.as_view(), name='students-list'),
```

Comment:
rename it back to pk please

Reply:
we will think about reneming all pk's later
[SCHOOL-658](https://lamart.atlassian.net/browse/SCHOOL-658)